python/tvm/relay/frontend/tensorflow.py

```diff
@@ -406,8 +425,10 @@ def _impl(inputs, attr, params):
         # NHWC
         attr['layout'] = 'NHWC'
 
+        # Ignore the new attribute `half_pixel_centers` from TF2.0, for now.
+        #   The semantics of the attribute is not very clear at this point.
         return AttrCvt(op_name="resize",
-                       ignores=['Tdim'],
+                       ignores=['Tdim', 'half_pixel_centers'],
```

Comment:
it would be nice if add `half_pixel_centers` in tvm resize op. If the result is different from tensorflow, we might need to raise error instead of ignoring

Reply:
I think half_pixel_centers is the new name of align_corners (used in TF 1.X) that we do support in tvm resize.

Reply:
then add `transforms={'half_pixel_centers':'align_corners'}` here should be working

Reply:
It seems the two attributes serve different purposes. The [doc](https://www.tensorflow.org/api_docs/cc/struct/tensorflow/ops/resize-bilinear/attrs) says nothing really. The [source code](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/kernels/resize_nearest_neighbor_op.cc#L147) does `half_pixel_centers` processing is applied optionally on top of `align_corners` processing.

Reply:
Idea: How about a unit test that compares a TF model using that attribute to Relay ignoring the attribute? I was reading the test suite to add to this PR, when I saw little coverage on such kind of issue, and fixing the tutorial seemed good to release early.

I'll try a test today, but if you have some guidance/advice/preference, I'd be happy to hear it. It may just be a different PR.